BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+        # Decrypt the ciphertext
+        plaintext = cipher.decrypt(ciphertext)
+
+        # Unpad the plaintext
+        unpadded_plaintext = unpad(plaintext, AES.block_size)
+
+        # Save the decrypted file
+        decrypted_file_path = file_path.replace(".enc", "")
+        with open(decrypted_file_path, "wb") as f:
+            f.write(unpadded_plaintext)
+
+        return f"File decrypted successfully: {decrypted_file_path}"
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)